ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master__okd-scos.yaml

```diff
@@ -26,17 +26,12 @@ images:
   to: cluster-samples-operator
 promotion:
   to:
-  - name: "4.17"
+  - name: scos-4.17
```

Comment:
the `ocp_builder_stream9` above should we replaced with the origin one:
```
 origin_scos-4.17_base-stream9:
    name: scos-4.17
    namespace: origin
    tag: base-stream9
```

Reply:
Thanks, this seems to have worked! 🥇